backend/auction/views.py

```diff
@@ -86,3 +87,39 @@ def delete(self, request, auction_id, format=None):
         auction = get_object_or_404(Auction, id=auction_id)
         auction.delete()
         return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+class AddToAuctionApiView(APIView):
+    """
+    Takes in a vehicle and auction ID and associates
+    it with an auction by creating an AuctionItem
+    """
+
+    def post(self, request):
```

Comment:
@kevinrczhang I think if the endpoint has the `auction_id` and `vehicle_id` in the endpoint URL, then we can update the method signature to take in `auction_id` and `vehicle_id`.